src/chainparams.cpp

```diff
@@ -204,8 +204,8 @@ class CTestNetParams : public CChainParams {
         consensus.vDeployments[Consensus::DEPLOYMENT_TAPROOT].nStartTime = 1199145601; // January 1, 2008
         consensus.vDeployments[Consensus::DEPLOYMENT_TAPROOT].nTimeout = 1230767999; // December 31, 2008
 
-        consensus.nMinimumChainWork = uint256S("0x0000000000000000000000000000000000000000000001495c1d5a01e2af8a23");
-        consensus.defaultAssumeValid = uint256S("0x000000000000056c49030c174179b52a928c870e6e8a822c75973b7970cfbd01"); // 1692000
+        consensus.nMinimumChainWork = uint256S("0x0000000000000000000000000000000000000000000001db6ec4ac88cf2272c6");
+        consensus.defaultAssumeValid = uint256S("0x000000000000006433d1efec504c53ca332b64963c425395515b01977bd7b3b0"); // 1864000
```

Comment:
Testnet nMinimumChainWork and defaultAssumeValid consensus params match up for me
```
$ ./src/bitcoin-cli -testnet getblockheader 000000000000006433d1efec504c53ca332b64963c425395515b01977bd7b3b0
{
  "hash": "000000000000006433d1efec504c53ca332b64963c425395515b01977bd7b3b0",
  "height": 1864000,
  "chainwork": "0000000000000000000000000000000000000000000001db6ec4ac88cf2272c6",
}
```